scalafix/rules/src/main/scala/fix/Monocle3.scala

```diff
@@ -0,0 +1,34 @@
+package fix
+
+import scalafix.v1._
+import scala.meta._
+
+class Monocle3 extends SemanticRule("Monocle") {
+
+  override def fix(implicit doc: SemanticDocument): Patch = {
+    SetRules(doc.tree)
+  }.asPatch
+
+  object SetRules {
+    private[this] val setMatcher =
+      SymbolMatcher.normalized("monocle/PLens#set.")
```

Comment:
How can we add other optics type? `PPrism`, `POptional`, `PTraversal` and `PSetter`

Reply:
Yes will get to it this week hopefully